examples/arithmetics/src/language-server/generated/meta-data.ts

```diff
@@ -0,0 +1,9 @@
+/******************************************************************************
+ * This file was generated by langium-cli 0.1.0.
+ * DO NOT EDIT MANUALLY!
+ ******************************************************************************/
+
+export class ArithmeticsLanguageMetaData {
+    languageId = 'arithmetics';
+    extensions = ['.calc'];
```

Comment:
Do we need the leading `.` ?  I saw some js API returns extensions wit a dot `Path.extname`, but some do not

Reply:
The `extensions` now always start with a leading dot.